js/binance.js

```diff
@@ -5006,4 +5006,26 @@ module.exports = class binance extends Exchange {
         }
         return result;
     }
+
+    async transferIn (code, amount, params = {}) {
+        // transfer from spot wallet to coinm or usdm futures wallet
+        let num = undefined;
+        if (this.options['defaultType'] === 'delivery') {
```

Comment:
The problem with this implementation is that `delivery` is binance-specific, we will remove that eventually. 

Most of the exchanges will have `future` = expiring futures, and `swap` for perpetual future swaps. 

Binance is an outlier which we have to take into account for historical reasons. Binance's old docs said `delivery` = expiring futures, and `future` = perpetual future swaps.

Reply:
This will be fixed, but the fix isn't trivial: https://github.com/ccxt/ccxt/pull/10720

Reply:
thanks, so I'll close this PR, right?

Reply:
Yes, i would suggest that we collect more exchange-specific implementation of this method, so that we could unify it having a wider overview of the common subset.